monai/networks/trt_compiler.py

```diff
@@ -255,6 +345,7 @@ def __init__(
                     'torch_trt' may not work for some nets. Also AMP must be turned off for it to work.
             input_names: Optional list of input names. If None, will be read from the function signature.
             output_names: Optional list of output names. Note: If not None, patched forward() will return a dictionary.
+            output_lists: Optional list of output lists.
```

Comment:
Could you please add more details about this parameter and the relation between this one and the output_names? Now it's hard to understand the meaning of it.

Thanks,  
Bin